### ISSUE TITLE
Handle process death.

### ISSUE THREAD

**Issue by guelo** [PandoraMedia/BottomNavigator#1]
Save BottomNavigator state in saveInstanceState.

**Comment by davidbilik** [comment]
What is the current behavior on process death? Default init settings? Isnt it a leak? Because fragments are still inside FragmentManager, arent they?

**Comment by guelo** [comment]
Yes it's a leak. BottomNavigator will create new fragments and ignore the existing ones inside FragmentManager.

**Comment by guelo** [comment]
Moved the memory leak bug to Issue #4. This ticket is for the enhancement to preserve BottomNavigator state after process death.
